kcsd/KCSD.py

```diff
@@ -672,10 +672,10 @@ def estimate_at(self):
         self.estm_x, self.estm_y : Locations at which CSD is requested.
 
         """
-        nx = (self.xmax - self.xmin)/self.gdx
-        ny = (self.ymax - self.ymin)/self.gdy
-        self.estm_pos = np.mgrid[self.xmin:self.xmax:np.complex(0, nx), 
-                                 self.ymin:self.ymax:np.complex(0, ny)]
+        nx = int(np.rint((self.xmax - self.xmin)/self.gdx)) + 1
+        ny = int(np.rint((self.ymax - self.ymin)/self.gdy)) + 1
+        self.estm_pos = np.meshgrid(np.linspace(self.xmin, self.xmax, nx),
+                                    np.linspace(self.ymin, self.ymax, ny))
```

Comment:
If the order of axis matters (see line 681), you need to use:
```suggestion
        self.estm_pos = np.meshgrid(np.linspace(self.xmin, self.xmax, nx),
                                    np.linspace(self.ymin, self.ymax, ny),
                                    indexing='ij')
```